dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -304,6 +306,20 @@ def diff(self, other_profile: CategoricalColumn, options: dict = None) -> dict:
                     other_profile._categories.items(), key=itemgetter(1), reverse=True
                 )
             )
+            if cat_count1.keys() == cat_count2.keys():
+                total_psi = 0.0
+                for key in cat_count1.keys():
+                    perc_A = cat_count1[key] / self.sample_size
+                    perc_B = cat_count2[key] / other_profile.sample_size
+                    total_psi += (perc_B - perc_A) * math.log(perc_B / perc_A)
+                    differences["statistics"]["psi"] = total_psi
```

Comment:
this really could be at the end of the for loop (missed that on the review on my phone in the merge to `dev`

Reply:
bit DRY-er... but not a huge functional issue